### ISSUE TITLE
loss：nan

### ISSUE THREAD

**Issue by lxyzler** [clovaai/deep-text-recognition-benchmark#132]
workers: 4
batch_size: 16
adam: False
lr: 1
beta1: 0.9
rho: 0.95
eps: 1e-08
grad_clip: 5
batch_ratio: 1
total_data_usage_ratio: 1.0
batch_max_length: 300
imgH: 32
imgW: 1024
rgb: True

[72000/5000000] valid loss: 1.01961 accuracy: 47.740, norm_ED: 60783.96
[74000/5000000] valid loss: 1.04614 accuracy: 47.155, norm_ED: 61910.29
[76000/5000000] valid loss: nan accuracy: 0.001, norm_ED: 369951.00

I'm training long text recognition models.
Before 7600, the training was normal. By 7600, loss became Nan. 
I tried to reduce the learning rate（Adadelta）, but I still couldn't. what's the reason

**Comment by jjprincess** [comment]
> workers: 4
> batch_size: 16
> adam: False
> lr: 1
> beta1: 0.9
> rho: 0.95
> eps: 1e-08
> grad_clip: 5
> batch_ratio: 1
> total_data_usage_ratio: 1.0
> batch_max_length: 300
> imgH: 32
> imgW: 1024
> rgb: True
> 
> [72000/5000000] valid loss: 1.01961 accuracy: 47.740, norm_ED: 60783.96
> [74000/5000000] valid loss: 1.04614 accuracy: 47.155, norm_ED: 61910.29
> [76000/5000000] valid loss: nan accuracy: 0.001, norm_ED: 369951.00
> 
> I'm training long text recognition models.
> Before 7600, the training was normal. By 7600, loss became Nan.
> I tried to reduce the learning rate（Adadelta）, but I still couldn't. what's the reason

Did you solve this problem?
